packages/analytics-js-integrations/src/integrations/GainsightPX/browser.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable lines-between-class-members */
+import {
+  NAME,
+  DISPLAY_NAME,
+} from '@rudderstack/analytics-js-common/constants/integrations/GainsightPX/constants';
+import Logger from '../../utils/logger';
+import { loadNativeSdk } from './nativeSdkLoader';
+import { getDestinationOptions } from './utils';
+
+const logger = new Logger(DISPLAY_NAME);
+
+class GainsightPX {
+  constructor(config, analytics, destinationInfo) {
+    if (analytics.logLevel) {
+      logger.setLogLevel(analytics.logLevel);
+    }
+    this.analytics = analytics;
+    this.productKey = !config.productKey ? '' : config.productKey;
```

Comment:
```suggestion
    this.productKey = config.productTagKey;
```

Reply:
@Gauravudia - I see in your comment, you are referring to "productTagKey", but in the config code, I am using "productKey".  Are you suggesting that I change them all to productTagKey?  Is the name significant?  Or was that a typo and you are just suggesting I get rid of the conditional assignment?